oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -830,6 +833,25 @@ public void IdTokenCredentials_NoUserEmailScope_throws() throws IOException {
     }
   }
 
+  @Test
+  public void UserCredentials_ToBuilder_CopyEveryAttributes() {
```

Comment:
nit, every part has to stat with lowercase, like this and a typo fix: userCredentials_toBuilder_copyEveryAttribute